packages/fxa-settings/src/components/DataBlock/index.tsx

```diff
@@ -43,46 +22,24 @@ export const DataBlock = ({
     onCopied && onCopied(copyValue);
   }
 
-  useEffect(() => {
-    !inputId && setInputId(uuidv4());
-  }, [inputId, id]);
-
-  const onClick = (event: MouseEvent<HTMLButtonElement>) => {
-    copyable && copyToClipboard();
-  };
-
   return (
-    <>
-      <button
-        className={`flex rounded-xl px-7 font-mono text-sm text-green-900 bg-green-800 bg-opacity-10 flex-wrap ${
-          valueIsArray ? 'py-4' : 'py-5'
-        } ${
-          copyable
-            ? 'hover:bg-opacity-20 focus:bg-opacity-30 active:bg-opacity-30'
-            : 'cursor-default select-text'
-        }`}
-        aria-describedby={readerText ? readerId : undefined}
-        id={inputId}
-        data-testid="datablock-button"
-        disabled={!copyable}
-        {...{ onClick }}
-      >
-        {valueIsArray ? (
-          (value as string[]).map((item) => (
-            <span key={item} className="flex-50% py-1">
-              {item}
-            </span>
-          ))
-        ) : (
-          <span>{value}</span>
-        )}
-      </button>
-      {readerText && (
-        <span data-testid="datablock-srtext" className="sr-only" id={readerId}>
-          {readerText}
-        </span>
+    <button
+      className={`flex rounded-xl px-7 font-mono text-sm text-green-900 bg-green-800 bg-opacity-10 flex-wrap hover:bg-opacity-20 focus:bg-opacity-30 active:bg-opacity-30 ${
+        valueIsArray ? 'py-4' : 'py-5'
```

Comment:
I think some of what what was done in this PR/the other could look a bit cleaner using `classNames` lib, but this works👍